### DISCUSSION TITLE
Is there a way to re-trigger x-teleport?

### DISCUSSION THREAD

**Discussion by docdunning** [alpinejs/alpine#4180] [1. Help]
I've used `x-teleport` to take two elements from their parents and have them in another part of the DOM.  They show up (as expected) in the same order as their parents.

But now I swap the positions of the parents in the DOM.  I'd like the two teleported elements to swap position as well, but they stay where they were.

I'm not entirely surprised by this, but I'm just wondering if there's a way to have Alpine regenerate the x-teleported elements so that they maintain the same order as their parents.

If not, I guess I just have to use some standard JS to move the teleported elements around.

**Comment by ekwoka** [comment]
You can change the attribute on it, or have the expression evaluate to something you can change.

**Reply by docdunning** [reply]
Thanks, I'll give that a try.

**Reply by docdunning** [reply]
> You can change the attribute on it, or have the expression evaluate to something you can change.

Sorry, I've not been able to get that to work.  No worries, I'll take a different approach that doesn't use `x-teleport`.
